test/es6-import.js

```diff
@@ -0,0 +1,12 @@
+import jsdomify from 'jsdomify';
+import {assert} from 'chai';
+import Cookies from '../src/js.cookie';
+
+it('should function properly when jsdom is used', () => {
```

Comment:
From what I understand, QUnit only runs in the browser. The problem doesn't happen in the browser, only when the code is run serverside in node.